_posts/bashar_allabadi/helm-charts-best-practices.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title:  "Helm for Kuberenetes, Lessons Learned"
+breadcrumb: true
+author: bashar_allabadi
+date: 2021-03-31
+categories: bashar_allabadi
+tags:
+    - Devops
+    - Helm
+    - Helm Charts
+    - Best Practice
+teaser:
+    info: Team's experience with helm, and the lessons we learnd 
+    image: bashar_allabadi/helm.svg
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+
+## Background
+
+We started the ICGC ARGO project (https://platform.icgc-argo.org/) in mid 2019, and it's still on going. so it's relatively new project and the team decided to adopt and use Kubernetes (https://kubernetes.io/) for the first time. 
```

Comment:
suggested grammar and links:

We started the [ICGC ARGO project](https://platform.icgc-argo.org/) in mid 2019, and it's still ongoing. It's still 
 a relatively new project and the team decided to adopt and use [Kubernetes](https://kubernetes.io/) for the first time.